app/shops.py

```diff
@@ -0,0 +1,8 @@
+class Shops:
+    def __init__(self,
+                 shop_name: str,
+                 shop_location: list,
```

Comment:
List of what? Add more descriptive typehint, please. For example, if It's a list of integers: list[int] and so on